### DISCUSSION TITLE
Logs Navigation: Timepicker interval does not correspond to interval in logs navigation

### DISCUSSION THREAD

**Discussion by cdalexndr** [grafana/grafana#39296] [Observability]
**What happened**:

For log batch timed `21:19:54 - 22:16:04` the top date picker incorrectly shows `19:16:04 - 22:16:04` meaning it includes the previous batch dated `18:57:18 - 21:19:54`, but it does NOT.
This is confusing.

![131262873-2a51d1df-6506-4ccf-9540-1f2ec74fc965](https://user-images.githubusercontent.com/22418250/131263043-706ef057-7c2d-4fc5-a0ff-7e6f28f2dd50.png)

**What you expected to happen**:
Correct time range.

**How to reproduce it (as minimally and precisely as possible)**:
1. search logs (ex: last X hours)
2. get `Older logs` to load previous batch
3. navigate back to the first log batch and notice incorrect date

**Anything else we need to know?**:

Related to  PR https://github.com/grafana/grafana/pull/33259 and feature https://github.com/grafana/grafana/issues/19354

**Environment**:
- Grafana version: Grafana v8.1.0 (62e720c06b)
- Data source type & version: Elasticsearch 7.10.1
- OS Grafana is installed on: docker
- User OS & Browser: Win 10 & Firefox
- Grafana plugins: default
- Others:

**Comment by ivanahuckova** [comment]
This is maybe more of a UX issue +cc @grafana/grafana-labs-ux-design because in time picker, we are showing the request range  - range query was run with. In logs navigation, we are showing visible range - the "from time" based on request and "to time" based on last received log line - the range of the results. So the visualised ranges are technically correct in both cases, but I can see how this could be confusing. Maybe @grafana/grafana-labs-ux-design have thoughts on how to solve this.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
